src/sickle.ts

```diff
@@ -156,6 +167,22 @@ class Annotator {
       this.writeRange(node.getFullStart(), node.getEnd());
     } else {
       switch (node.kind) {
+        case ts.SyntaxKind.ExportDeclaration:
+          let exportDecl = <ts.ExportDeclaration>node;
+          if (!exportDecl.exportClause && exportDecl.moduleSpecifier) {
+            // It's an "export * from ..." statement.
```

Comment:
Can't help but wonder what happens when you have circular 'export *',

a.ts
export * from 'b'

b.ts
export * from 'a'
export var b;

No action just wondering if you tested that.